pkgs/development/python-modules/nbclient/default.nix

```diff
@@ -1,9 +1,19 @@
-{ lib, buildPythonPackage, fetchPypi, pythonOlder,
-  async_generator, traitlets, nbformat, nest-asyncio, jupyter-client,
-  pytest, xmltodict, nbconvert, ipywidgets
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, async_generator
+, traitlets
+, nbformat
+, nest-asyncio
+, jupyter-client
+, pytestCheckHook
+, xmltodict
+, nbconvert
+, ipywidgets
 }:
 
-buildPythonPackage rec {
+let nbclient = buildPythonPackage rec {
```

Comment:
Why not use https://github.com/NixOS/nixpkgs/pull/119942?

Reply:
It's a big change just before branch-off that has performance implications. It won't surprise me if it will be reverted. Also, when we want to use it we should do that in `buildPython*`.

Reply:
And I didn't know how to get the package attribute. Just oldAttr maybe?